common/src/main/java/tachyon/util/CommonUtils.java

```diff
@@ -66,7 +96,7 @@ public static void sleepMs(Logger logger, long timeMs, boolean shouldInterrupt)
   }
 
   /**
-   * Common empty loop utilities to warm up
+   * Common empty loop utilities to warm up.
    */
   public static void warmUpLoop() {
     for (int k = 0; k < 10000000; k ++) {
```

Comment:
Could you add some additional info to the javadoc, like why the empty loop helps the JVM warm up, etc. I think we should keep it since we try to include some microbenchmarking in Tachyon. If we end up moving that out of Tachyon or to another module, I could see getting rid of this.